internal/cli/config.go

```diff
@@ -18,19 +18,46 @@ var (
 
 // Config is botkube cli config
 type Config struct {
-	Token string `json:"token"`
+	Token     string          `json:"token"`
+	Telemetry TelemetrySwitch `json:"telemetry"`
 }
 
-// NewConfig creates new Config from local data
-func NewConfig() (*Config, error) {
+// TelemetrySwitch controls telemetry
+type TelemetrySwitch string
+
+const (
+	// TelemetryEnabled is a value for enabled telemetry
+	TelemetryEnabled TelemetrySwitch = "enabled"
```

Comment:
This value is not used at all. Does it make sense to use enum instead of `bool`? I know there could be multiple levels of telemetry at some point, but for now it looks like an overkill to me 🤔

Reply:
I forgot to include it in telemetry/enable.go. It is used now.

Reply:
ok 👍